test_haystack/solr_tests/test_management_commands.py

```diff
@@ -3,20 +3,26 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
 
 import datetime
+import pysolr
+import os
 from tempfile import mkdtemp
+try:
+    from StringIO import StringIO
```

Comment:
n.b. this will require reconciling usage with `io.StringIO` being text (and thus Unicode) vs. binary.